src/classes/issues-processor.ts

```diff
@@ -268,6 +317,130 @@ export class IssuesProcessor {
       );
     }
 
+    if (this.options.mlflow) {
+      const {startDate, daysBeforeReplyReminder, daysBeforeAssigneeReminder} =
+        this.options;
+      if (
+        issue.milestone ||
+        issue.isStale ||
+        (startDate &&
+          !isDateMoreRecentThan(
+            new Date(issue.created_at),
+            new Date(startDate)
+          ))
+      ) {
+        return;
+      }
+
+      const reminderToMaintainers = 'Reminder to MLflow maintainers';
+      if (issue.isPullRequest) {
+        // TODO
+      } else {
+        const hasClosingPr = issue.labels.some(
+          ({name}) => name === 'has-closing-pr'
+        );
+        if (hasClosingPr) {
+          issueLogger.info('This issue has a closing PR');
+          return;
+        }
```

Comment:
Does this bot post comments / reminders for stale PRs? If not, in the future, the bot should also ping maintainers / PR filers if a PR: 1. hasn't been reviewed by a maintainer or 2. hasn't had comments resolved by the filer.

Reply:
Ah, I see: `if (issue.isPullRequest)` at https://github.com/harupy/stale/pull/1/files#diff-93e32a46e13f11c2066be36506b0f153d7da2b6b8d787da9177c10692ce50fc5R336-R338. Were you planning to add PR support to this PR or do it as a follow-up some time in the future? Future sounds fine to me :)

Reply:
I was planning to do it as a follow-up.